### ISSUE TITLE
Support for rented vs. owned assets

### ISSUE THREAD

**Issue by coolbung** [grokability/snipe-it#8478]
**Server (please complete the following information):**
 - Snipe-IT Version : 4.9.4
 - OS: Ubuntu
 - Web Server: Apache
 - PHP Version : 7.2

**Is your feature request related to a problem? Please describe.**
Often, companies rent assets to reduce capital costs. Didn't see a "native" way of managing the rental lifecycle in SnipeIT

**Describe the solution you'd like**
- A flag (checkbox) to mark if an asset is rented
- A field to define the rental period (weekly/monthly/quarterly/annual)
- Rental cost for that period
- Deposit payment made to the vendor
- For any change in the rental cost, an existing asset can be marked as archived and a new one created

**Describe alternatives you've considered**
- Creating 2 companies - one for the rented assets another for owned
- Custom fields (however the fieldset needs to be added to each model)

**Additional context**
none

**Comment by ggiesen** [comment]
ping

**Comment by FreeMinded** [comment]
Would be nice if an asset could have an owner which is independent from the user/company. The field owner could just be a different reference to the already existing company list.

**Comment by snipe** [comment]
I don't see a lot of people asking for this, and it could just as easily be handled via custom fields and/or notes IMHO

**Comment by davidebonavita** [comment]
I would add a +1 here, custom fields would suffice but I don't think we can have notifications about an expiring lease through those

**Comment by FreeMinded** [comment]
@snipe I agree with the custom field option is a viable workaround. But it would be nice if the custom field could reference a company from the corresponding table.

**Comment by Louson** [comment]
Hello ! It does not look sufficient to me. Custom fields are only for assets, aren't they ?

In our case, we have some assets and accessories which are owned by a client. We would like to track them but we don't want to mix them with our stuff.

**Comment by ts-biotech** [comment]
This feature is crucial for calculation costs on assets and accessories. Rental and leasing is very much used in the IT industry.
